style.css

```diff
@@ -0,0 +1,25 @@
+.app-header {
```

Comment:
some classes here are not needed and you could have used bootstrap for them
example mt-5 (margin-top) and w-auto (fit content)